scripts/build-ghp-index.js

```diff
@@ -0,0 +1,100 @@
+  </td>
+  <td class="overflow-hidden whitespace-nowrap px-4 py-3 text-ellipsis">${symbol}</td>
+  <td class="overflow-hidden whitespace-nowrap px-4 py-3 text-ellipsis">${chainId === 43111 ? "Hemi" : "Hemi Sepolia"}</td>
+  <td class="px-4 py-3">
+    <a class="font-mono text-neutral-500 hover:text-neutral-700" href="https://${chainId === 43111 ? "explorer" : "testnet.explorer"}.hemi.xyz/token/${address}" title="${address}">
+      ${shortenAddress(address)}
+    </a>
+  </td>
+</tr>
+`;
+
+const page = ({ name, tokens }) => `
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="description" content="List of tokens on the Hemi networks.">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>${name}</title>
+
+  <link rel="preload" href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600&family=JetBrains+Mono:wght@400&display=swap" as="style" />
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600&family=JetBrains+Mono:wght@400&display=swap" rel="stylesheet" />
+
+  <script src="https://cdn.tailwindcss.com"></script>
+  <script>
+    tailwind.config = {
+      theme: {
+        extend: {
+          fontFamily: {
+            mono: ['JetBrains Mono'],
+            sans: ['Inter']
+          }
+        }
+      }
+    }
+  </script>
+</head>
+
+<body class="bg-zinc-50 px-2 text-neutral-950">
+  <header class="flex items-center justify-between max-w-5xl mx-auto py-2.5 border-b border-gray-200">
+    <a href="https://hemi.xyz">
+      <img src="assets/hemi-logo.svg" alt="Hemi" height="28" width="78" />
+    </a>
+    <span>
+      <a class="ml-3 px-4 py-2 rounded-lg border-t border-gray-200 shadow-md" href="https://docs.hemi.xyz">Read the docs</a>
+      <a class="ml-3 px-4 py-2 rounded-lg text-white bg-orange-500 shadow-md" href="https://app.hemi.xyz">Launch Portal</a>
+    </span>
+  </header>
+  <section class="flex flex-col items-center">
+    <img class="my-12" src="assets/hemi-logo-orange.svg" height="131" width="132" alt="Hemi" />
+    <h1 class="max-w-xl text-5xl text-center font-semibold">Explore tokens on the Hemi networks</h1>
+    <table class="max-w-full overflow-hidden mt-14 mb-20 rounded-xl shadow-md">
+      <thead class="hidden lg:table-header-group h-11 border-b bg-zinc-100">
+        <tr>
+          <th class="px-4 py-3 text-left font-normal">Name</th>
+          <th class="px-4 py-3 text-left font-normal">Symbol</th>
+          <th class="px-4 py-3 text-left font-normal">Chain</th>
+          <th class="px-4 py-3 text-left font-normal">Address</th>
+        </tr>
+      </thead>
+      <tbody>
+        ${tokens
+          .sort((a, b) => a.name.localeCompare(b.name))
+          .sort((a, b) => a.chainId - b.chainId)
+          .map((token) => row(token))
+          .join("")}
+      </tbody>
+    </table>
+  </section>
+</body>
+
+</html>
+`;
```

Comment:
my eyessss  reading this without proper coloring 😆  I just hope we don't have to update this ever

Reply:
We can't code in green-screens like in the 80s anymore.